netbsd/Platform.c

```diff
@@ -254,11 +254,12 @@ double Platform_setCPUValues(Meter* this, int cpu) {
       v[CPU_METER_IOWAIT]  = 0.0;
       v[CPU_METER_FREQUENCY] = NAN;
       this->curItems = 8;
-      totalPercent = v[0] + v[1] + v[2] + v[3];
+      totalPercent = v[CPU_METER_NICE] + v[CPU_METER_NORMAL] + v[CPU_METER_KERNEL] + \
+         v[CPU_METER_IRQ];
```

Comment:
No word-wrapping long lines. Also no backslash for line continuation outside macros.